cibuild.sh

```diff
@@ -289,7 +289,7 @@ function run_builds {
   options+="-j $ncpus"
 
   for build in $builds; do
-    $nuttx/tools/testbuild.sh $options $build
+    $nuttx/tools/testbuild.sh $options -e -Wno-cpp $build
```

Comment:
Should we add "" around -Wno-cpp

Reply:
Now it may be well without "" around since there is only -Wno-cpp flag specified. If more flags specified, we should add "" around such as "-Wno-cpp -Werror".